File: qlty-cli/tests/lang/php/basic.in/functions/methods_with_parameters.php
<?php
  class Foo {
    public $bar;
    public $baz;

    public function __construct() {
      $this->bar = "";
      $this->baz = "";
    }

    public function doSomething($baz, $bar) {
      $this->bar = $bar;
      $this->baz = $baz;
      return ($this->bar . $this->baz);
    }
  }
?>

File: qlty-cli/tests/lang/go/basic.in/functions/methods_with_parameters.go
type Foo struct {
	dog string
	cat string
}

func NewFoo(dog, cat string) *Foo {
	return &Foo{
		dog: "Ruff",
		cat: "Meow",
	}
}

func (f *Foo) Bar(dog, cat interface{}) []string {
	return []string{toString(dog), toString(cat)}
}
